server/server.go

```diff
@@ -61,6 +62,7 @@ type HTTPServer struct {
 	Config         Configuration
 	ServicesConfig services.Configuration
 	GroupsChannel  chan []groups.Group
+	amsClient      *amsclient.AMSClient
```

Comment:
is it lowercase by mistake?

Reply:
No, it is just to don't expose it, as it shouldn't be used directly by any uer of the HTTPServer objects. It can be changed, of course